# Cell-engineering-Regeneration
*****************

**Table of Contents**:

- [cell_imaging](#cell_imaging)
  - [image processing(segmentation/detection/classification)](#image_processing(segmentation/detection/classification))
- [single-cell_omics](#single-cell_omics)
- [cell](#cell)
*****************


## cell imaging
### image processing(segmentation/detection/classification)
1.[ESC-Track: A computer workflow for 4-D segmentation, tracking, lineage tracing and dynamic context analysis of ESCs](https://pubmed.ncbi.nlm.nih.gov/32720710/).<br>
2.[Automatic three‐dimensional segmentation of mouse embryonic stem cell nuclei by utilising multiple channels of confocal fluorescence images](https://onlinelibrary.wiley.com/doi/10.1111/jmi.12949).<br>
3.[Evaluation of Deep Learning Strategies for Nucleus Segmentation in Fluorescence Images](https://onlinelibrary.wiley.com/doi/full/10.1002/cyto.a.23863).<br>
4.[DeLTA: Automated cell segmentation, tracking, and lineage reconstruction using deep learning](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1007673).<br>
5.[ChipSeg: an automatic tool to segment bacteria and mammalian cells cultured in microfluidic devices](https://www.biorxiv.org/content/10.1101/2020.08.03.225045v1.abstract).<br>
6.[A Rapid and Efficient 2D/3D Nuclear Segmentation Method for Analysis of Early Mouse Embryo and Stem Cell Image Data](https://www.sciencedirect.com/science/article/pii/S2213671114000277#fig1).<br>
7.[Survey statistics of automated segmentations applied to optical imaging of mammalian cells](https://link.springer.com/article/10.1186/s12859-015-0762-2).<br>
8.[Robust and automated three-dimensional segmentation of densely packed cell nuclei in different biological specimens with Lines-of-Sight decomposition](https://link.springer.com/article/10.1186/s12859-015-0617-x).<br>
9.[Automated Identification and Localization of Hematopoietic Stem Cells in 3D Intravital Microscopy Data](https://www.sciencedirect.com/science/article/pii/S2213671115001605).<br>
10.[A high-throughput imaging and nuclear segmentation analysis protocol for cleared 3D culture models](https://www.nature.com/articles/s41598-018-29169-0).<br>
11.[Segmentation of Nuclei From 3D Microscopy Images of Tissue via Graphcut Optimization](https://ieeexplore.ieee.org/abstract/document/7346405).<br>
12.[3D Clumped Cell Segmentation Using Curvature Based Seeded Watershed](https://www.mdpi.com/2313-433X/2/4/31).<br>
13.[A method for the evaluation of thousands of automated 3D stem cell segmentations](https://onlinelibrary.wiley.com/doi/abs/10.1111/jmi.12303?casa_token=OvrlH5uEexoAAAAA:T88Gs1Emcs8T4BaqOl9AwCBxWKUQTtXLhr9WL2LJhTGCP-MtrYKJlfTbjZAQVs5GpUuo710JUMLBHaY).<br>
14.[Brain development: machine learning analysis of individual stem cells in live 3D tissue](https://www.biorxiv.org/content/10.1101/137406v2.abstract).<br>
15.[NesSys: a novel method for accurate nuclear segmentation in 3D](https://www.biorxiv.org/content/10.1101/502872v1.abstract).<br>
16.[Three-dimensional GPU-accelerated active contours for automated localization of cells in large images](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0215843).<br>
17.[Software tools for 3D nuclei segmentation and quantitative analysis in multicellular aggregates](https://www.sciencedirect.com/science/article/pii/S2001037020302853).<br>
18.[Segmentation of cell nuclei in fluorescence microscopy images: An integrated framework using level set segmentation and touching-cell splitting](https://www.sciencedirect.com/science/article/abs/pii/S0031320316300280).<br>
19.[Nessys: A new set of tools for the automated detection of nuclei within intact tissues and dense 3D cultures](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.3000388&rev=1).<br>
20.[DeepSynth: Three-dimensional nuclear segmentation of biological images using neural networks trained with synthetic data](https://www.nature.com/articles/s41598-019-54244-5).<br>
21.[3D-Cell-Annotator: an open-source active surface tool for single-cell segmentation in 3D microscopy images](https://academic.oup.com/bioinformatics/article/36/9/2948/5709038?login=true).<br>
22.[GIANI: open-source software for automated analysis of 3D microscopy images](https://www.biorxiv.org/content/10.1101/2020.10.15.340810v1.abstract).<br>
23.[WaveletSEG: Automatic wavelet-based 3D nuclei segmentation and analysis for multicellular embryo quantification](https://www.biorxiv.org/content/10.1101/2020.07.24.220285v1.abstract).<br>
24.[TANGO: a generic tool for high-throughput 3D image analysis for studying nuclear organization](https://academic.oup.com/bioinformatics/article/29/14/1840/231770?login=true).<br>\
25.[CellSegm - a MATLAB toolbox for high-throughput 3D cell segmentation](https://link.springer.com/article/10.1186/1751-0473-8-16).<br>
26.[A deep convolutional neural network for classification of red blood cells in sickle cell anemia](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1005746).<br>
27.[A Manual Segmentation Tool for Three-Dimensional Neuron Datasets](https://www.frontiersin.org/articles/10.3389/fninf.2017.00036/full).<br>
28.[The Allen Cell Structure Segmenter: a new open source toolkit for segmenting 3D intracellular structures in fluorescence microscopy images](https://www.biorxiv.org/content/10.1101/491035v1.abstract).<br>
29.[Deep learning guided image-based droplet sorting for on-demand selection and analysis of single cells and 3D cell cultures](https://pubs.rsc.org/ko/content/articlelanding/2020/lc/d0lc00055h/unauth#!divAbstract).<br>
30.[Cellpose: a generalist algorithm for cellular segmentation](https://www.nature.com/articles/s41592-020-01018-x).<br>
31.[Deep learning for cell image segmentation and ranking](https://www.sciencedirect.com/science/article/abs/pii/S089561111830048X?casa_token=MoxaBeyOZ4kAAAAA:zq6Ytq-LT_djzcs6OT_-xp67U1JnNkD5fBJho_MKkHdvIfSoKli8Uh1h9lUGQmX-5ZQgnekTwQ4).<br>
32.[Cell segmentation methods for label-free contrast microscopy: review and comprehensive comparison](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-019-2880-8).<br>
33.[Split and merge watershed: A two-step method for cell segmentation in fluorescence microscopy images](https://www.sciencedirect.com/science/article/abs/pii/S1746809419301491?casa_token=I1PcXkLroL4AAAAA:cdjC-gaAP9XTpUBtG0ixS39dUlNEGHTOJqYid320RMEhbqiMIrIkvxNbre9o6y1vwmG9hz6TP3A).<br>
34.[Deep learning for cellular image analysis](https://www.nature.com/articles/s41592-019-0403-1).<br>
35.[CellProfiler 3.0: Next-generation image processing for biology](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.2005970).<br>
36.[Data-analysis strategies for image-based cell profiling](https://www.nature.com/articles/nmeth.4397).<br>
37.[Keras R-CNN: library for cell detection in biological images using deep neural networks](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-020-03635-x).<br>
38.[Oufti: an integrated software package for high‐accuracy, high‐throughput quantitative microscopy analysis](https://onlinelibrary.wiley.com/doi/full/10.1111/mmi.13264).<br>
39.[Deep Learning Automates the Quantitative Analysis of Individual Cells in Live-Cell Imaging Experiments](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1005177).<br>
40.[Label‐Free Identification of White Blood Cells Using Machine Learning](https://onlinelibrary.wiley.com/doi/full/10.1002/cyto.a.23794).<br>
41.[Label-free assessment of red blood cell storage lesions by deep learning](https://www.biorxiv.org/content/10.1101/256180v2.abstract).<br>
42.[OpenSegSPIM: a user-friendly segmentation tool for SPIM data](https://academic.oup.com/bioinformatics/article/32/13/2075/1743029?login=true).<br>
43.[A practical guide to intelligent image-activated cell sorting](https://www.nature.com/articles/s41596-019-0183-1).<br>
44.[Lineage Tracing: Computational Reconstruction Goes Beyond the Limit of Imaging](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6399003/).<br>
45.[Quanti.us: a tool for rapid, flexible, crowd-based annotation of images](https://www.nature.com/articles/s41592-018-0069-0).<br>
46.[ilastik: interactive machine learning for (bio)image analysis](https://www.nature.com/articles/s41592-019-0582-9).<br>
47.[Quantitative Digital Microscopy with Deep Learning](https://arxiv.org/abs/2010.08260).<br>
48.[U-Net: deep learning for cell counting, detection, and morphometry](https://www.nature.com/articles/s41592-018-0261-2).<br>

collection-[Deep learning in microscopy](https://www.nature.com/collections/cfcdjceech).<br>
## single-cell omics
1.[Lineage tracing meets single-cell omics: opportunities and challenges](https://www.nature.com/articles/s41576-020-0223-2).<br>
2.[Integrative single-cell analysis](https://www.nature.com/articles/s41576-019-0093-7).<br>
3.[CellTagging: combinatorial indexing to simultaneously map lineage and identity at single-cell resolution](https://www.nature.com/articles/s41596-019-0247-2).<br>
4.[Unravelling cellular relationships during development and regeneration using genetic lineage tracing](https://www.nature.com/articles/s41580-019-0186-3).<br>
5.[Single cell RNA-sequencing reveals cellular heterogeneity and trajectories of lineage specification during murine embryonic limb development](https://www.sciencedirect.com/science/article/pii/S0945053X19304020?casa_token=9QOyoYbHgrAAAAAA:9Pz8k5IefrLrVdNa7WZ9onIVk3XtqVkcpv8EuVO8-LgTlbnNHi__zc1UUVbHVQvl7VIq1bCcF0E).<br>
6.[Single-Cell Analysis of Regional Differences in Adult V-SVZ Neural Stem Cell Lineages](https://www.sciencedirect.com/science/article/pii/S2211124718319740).<br>
7.[Dissecting Cell Lineage Specification and Sex Fate Determination in Gonadal Somatic Cells Using Single-Cell Transcriptomics](https://www.sciencedirect.com/science/article/pii/S2211124719302487).<br>
8.[DNA barcodes evolve for high-resolution cell lineage tracing](https://www.sciencedirect.com/science/article/abs/pii/S1367593119300274?casa_token=HxmlxW_70z4AAAAA:q_P8SiLgQyP3RAqDgWuiBokyMoy1fCe04lLqSiO_k7Vv1j1EoHKNmsYkezh3yjn2a3XdreHpa78).<br>
9.[Recording development with single cell dynamic lineage tracing](https://dev.biologists.org/content/146/12/dev169730.abstract).<br>
10.[CALISTA: Clustering and Lineage Inference in Single-Cell Transcriptional Analysis](https://www.biorxiv.org/content/10.1101/257550v3.abstract).<br>
11.[Single-Cell Multi-omic Integration Compares and Contrasts Features of Brain Cell Identity](https://www.sciencedirect.com/science/article/pii/S0092867419305045#!).<br>
12.[The emergence of transcriptional identity in somatosensory neurons](https://www.nature.com/articles/s41586-019-1900-1).<br>
13.[Single-cell analysis of olfactory neurogenesis and differentiation in adult humans](https://www.nature.com/articles/s41593-020-0587-9).<br>
14.[Next-Generation Lineage Tracing and Fate Mapping to Interrogate Development](https://www.sciencedirect.com/science/article/abs/pii/S1534580720308418).<br>
15.

